src/script/page/CenterColumn/CenterColumn.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {FC} from 'react';
+
+import HistoryExport from 'Components/HistoryExport';
+import ConnectRequests from 'Components/ConnectRequests';
+import ConversationList from 'Components/Conversation';
+import GroupCreationModal from 'Components/Modals/GroupCreation/GroupCreationModal';
+
+import {registerReactComponent, useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {t} from 'Util/LocalizerUtil';
+
+import MainContent from '../MainContent';
+import RootProvider from '../RootProvider';
+
+import {ContentViewModel} from '../../view_model/ContentViewModel';
+
+const statesTitle = {
+  [ContentViewModel.STATE.CONNECTION_REQUESTS]: t('accessibility.headings.connectionRequests'),
+  [ContentViewModel.STATE.CONVERSATION]: t('accessibility.headings.conversation'),
+  [ContentViewModel.STATE.HISTORY_EXPORT]: t('accessibility.headings.historyExport'),
+  [ContentViewModel.STATE.HISTORY_IMPORT]: t('accessibility.headings.historyImport'),
+};
+
+interface CenterColumnProps {
+  contentViewModel: ContentViewModel;
+}
+
+const CenterColumn: FC<CenterColumnProps> = ({contentViewModel}) => {
+  const {state} = useKoSubscribableChildren(contentViewModel, ['state']);
+  const {initialMessage} = contentViewModel;
+
+  const teamState = contentViewModel.getTeamState();
+  const userState = contentViewModel.getUserState();
+
+  const title = statesTitle[state];
+
+  return (
+    <RootProvider value={contentViewModel}>
+      <MainContent contentViewModel={contentViewModel} />
+
+      <h1 className="visually-hidden">{title}</h1>
+
+      {state === ContentViewModel.STATE.CONNECTION_REQUESTS && (
+        <ConnectRequests teamState={teamState} userState={userState} />
+      )}
+
+      {state === ContentViewModel.STATE.CONVERSATION && (
+        <ConversationList initialMessage={initialMessage} teamState={teamState} userState={userState} />
+      )}
+
+      {state === ContentViewModel.STATE.HISTORY_EXPORT && <HistoryExport userState={userState} />}
+
+      <GroupCreationModal userState={userState} teamState={teamState} />
+
+      <div className="center-column__overlay" />
+    </RootProvider>
+  );
+};
```

Comment:
Why having both `CenterColumn` and `MainContent`. 
Everything should probably go to `MainContent` and it handles all the `ContentState`. WDYT?